codespell_lib/data/dictionary.txt

```diff
@@ -4123,6 +4123,7 @@ bootstap->bootstrap
 bootstapped->bootstrapped
 bootstapping->bootstrapping
 bootstaps->bootstraps
+booundary->boundary
```

Comment:
Mind adding this?
https://github.com/search?q=booundaries&type=code
```text
booundaries->boundaries
```

Reply:
30b3c38